model/gen-ai/events.yaml

```
@@ -38,6 +151,33 @@ groups:
     brief: >
       This event describes the tool or function response message.
```

```suggestion
      This event describes the response from a tool or function call passed inside the prompt to the GenAI model.
```
Clear that this is passed to the model, not received from it.

---

fixed in https://github.com/open-telemetry/semantic-conventions/pull/1469/commits/557b99dc09eb08565c501460a1fada716ed050e4 with the same caveat about "passed inside the prompt"